docs/4.4/enterprise/workflow/index.md

```diff
@@ -129,6 +129,87 @@ deny:
     roles: ['admin']
 ```
 
+## Adding a reason to approval requests
+
+Teleport 4.4.4 introduced the ability for users to request additional roles. `tctl`
+or the Access Workflows API makes it easy to dynamically approve or deny these requests.
+
+By using notes you can provide users with an unprivileged state and must
```

Comment:
Sentence is unclear, by `notes` did you mean `access requests`?

Reply:
I clarified this to be request reason. In early versions we used notes but it wasn't as flexible.